src/metabase/api/collection.clj

```diff
@@ -551,7 +551,18 @@
                    [:u.email :last_edit_email]
                    [:u.first_name :last_edit_first_name]
                    [:u.last_name :last_edit_last_name]
-                   [:r.timestamp :last_edit_timestamp]]
+                   [:r.timestamp :last_edit_timestamp]
+                   [{:select   [:status]
+                     :from     [:moderation_review]
+                     :where    [:and
+                                [:= :moderated_item_type "dashboard"]
+                                [:= :moderated_item_id :d.id]
+                                [:= :most_recent true]]
+                                ;; limit 1 to ensure that there is only one result but this invariant should hold true, just
+                                ;; protecting against potential bugs
+                     :order-by [[:id :desc]]
+                     :limit    1}
+                    :moderated_status]]
```

Comment:
Seems like this should be a join with `moderated_status` instead of a subquery. I can make the change for you